x-pack/plugins/enterprise_search/common/types/connectors.ts

```diff
@@ -160,15 +160,25 @@ export enum TriggerMethod {
   SCHEDULED = 'scheduled',
 }
 
+export enum SyncJobType {
+  FULL = 'full',
+  INCREMENTAL = 'incremental',
+  ACCESS_CONTROL = 'access',
```

Comment:
```suggestion
  ACCESS_CONTROL = 'access_control',
```

See [Possible values for job type](https://github.com/elastic/connectors-python/blob/main/docs/CONNECTOR_PROTOCOL.md)